### PR TITLE
Using bootstrap to open iframe

### PR DESCRIPTION
**Pull request by alex-deaconu**
Removed old unused code for storeAccountModal
Refactored app-subscription-status directive

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. These changes are staged at https://apps-stage-3.risevision.com/.

See corresponding CH and Apps PRs.

Thanks!

File: src/dtv-app-subscription-status.js [hunk 1]
```diff
@@ -44,65 +41,33 @@
...
-            }
-          });
-
-          var watchAccount = $scope.$watch("showStoreAccountModal", function(show) {
+          $scope.$watch("showStoreModal", function(show) {
             if (show) {
-              initStoreAccountModal();
-
-              watchAccount();
+              if (show) {
```
Comment (ezequielc) [inline src/dtv-app-subscription-status.js]: @alex-deaconu I think `if (show)` is checked twice by mistake. Thanks

Comment (alex-deaconu) [inline src/dtv-app-subscription-status.js]: @ezequielc oops, good catch. Will fix. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu I think I found an issue: when I click in Subscribe Now button and close Store, if I click Subscribe Now again, nothing happens, Store does not open. Could you check as well? Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc okay, I will take a look. Thanks.

File: src/dtv-subscription-status.js [hunk 2]
```diff
@@ -63,41 +61,34 @@
...
-              var angularDomEl = angular.element("<div store-modal></div>");
-              angularDomEl.attr({
-                "id": "store-modal",
-                "animate": "animate",
-                "show-store-modal": "showStoreModal",
-                "company-id": "{{companyId}}",
-                "product-id": "{{productId}}"
+              })
+              .finally(function() {
+                $scope.showStoreModal = false;
```
Comment (alex-deaconu) [inline src/dtv-subscription-status.js]: @ezequielc this fixes the issue where the modal doesn't open the second time. Also updated unit tests for this variable. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu :+1: Thanks for fixing!

**Pull request merged**